pages/common/sphinx-build.md

```diff
@@ -0,0 +1,16 @@
+# sphinx-build
+
+> Generates documentation from the files in sourcedir and places it in the outputdir.
```

Comment:
```suggestion
> Generates documentation from the files in sourcedir and places it in the outputdir.
> Sphinx Python documentation generator.
```
I would suggest rephrasing this to give a simpler and clearer description.

Reply:
I don’t think sphinx is specific to Python? It’s a general documentation generator, which uses Python to do so. So just `Sphinx documentation generator`?

Reply:
You're right. It can be used for multiple languages, not only Python. Sorry about that.